packages/Webkul/Ui/src/Resources/assets/js/components/datagrid/datagrid-plus.vue

```diff
@@ -640,7 +640,7 @@
                                                       'undefined' &&
                                                   action.function
                                                     ? action.function
-                                                    : doAction($event)
+                                                    : doAction($event, typeof action.confirm_text !== 'undefined' && action.confirm_text ? action.confirm_text : null)
```

Comment:
This should be clean as previously and this is not needed because the current code is written from the old Datagrid point of view. As you see the blade file `render()` method is used.